corehq/apps/data_interfaces/management/commands/get_case_rule_submissions.py

```diff
@@ -0,0 +1,83 @@
+import csv
+from datetime import datetime
+
+from django.core.management.base import BaseCommand, CommandError
+
+from corehq.apps.data_interfaces.models import CaseRuleSubmission
+from corehq.util.argparse_types import date_type
+from corehq.util.log import with_progress_bar
+
+
+class Command(BaseCommand):
+    help = "Output form IDs for forms created by case rules"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'domains',
+            metavar='domain',
+            nargs='+',
+        )
+
+        parser.add_argument(
+            '--rule-id',
+            type=int,
+            default=None,
+            help='Limit output to the rule with this ID only.'
+        )
+
+        parser.add_argument(
+            '--rule-name',
+            default=None,
+            help='Limit output to the rule with this name only.'
+        )
+
+        parser.add_argument('-a', '--archived', action='store_true', help='Only include archived forms.')
+        parser.add_argument('-s', '--startdate', type=date_type, help='Start date, inclusive. Format YYYY-MM-DD')
+        parser.add_argument('-e', '--enddate', type=date_type, help='End date, exclusive. Format YYYY-MM-DD')
+
+    def handle(self, *args, **options):
+        domains = options['domains']
+        rule_id = options['rule_id']
+        rule_name = options['rule_name']
+        if rule_id and rule_name:
+            raise CommandError("Specify either 'rule-id' or 'rule-name' but not both.")
```

Comment:
Is this to eliminate ambiguity, such as a situation where there is a valid `rule_id` and a valid `rule_name` that reference different rules?

Reply:
Yes. The alternative is to check that `rule_name` matches the rule referenced by `rule_id`